src/config.rs

```diff
@@ -163,13 +163,18 @@ impl Default for RawUpdatesConfig {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 struct RawDirectoriesConfig {
+    #[serde(default)]
+    pub cache_dir: Option<PathBuf>,
     #[serde(default)]
     pub custom_pages_dir: Option<PathBuf>,
 }
 
 impl Default for RawDirectoriesConfig {
     fn default() -> Self {
         Self {
+            cache_dir: None,
+            // FIXME: should this "business-logic" really be in `*Raw*DirectoriesConfig`?
+            // To me, this seems odd, especially since it can still be `None` afterwards 🤔
```

Comment:
You're right. This would be addressed in the latest commit in #276 ("Move custom_pages_dir business logic from RawConfig to Config").